<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogSoft</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushCoreTakenButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Push core taken</string>
    </property>
   </widget>
   <widget class="QPushButton" name="niskinBottleClosedButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Niskin bottle closed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transectVideoStartButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Transect video start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transectVideoStopButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Transect video stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeDataLogButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastEventLogTextbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>355</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lastEventLogLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Last event logged</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastTimeLogTextbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>385</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lastTimeLogLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Last time logged</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sampleTakenButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample taken</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otherEventButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Other event</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
